src/main/java/bio/terra/service/snapshot/flight/delete/DeleteSnapshotAuthzBqAcls.java

```diff
@@ -0,0 +1,72 @@
+package bio.terra.service.snapshot.flight.delete;
+
+import bio.terra.service.iam.AuthenticatedUserRequest;
+import bio.terra.service.iam.IamResourceType;
+import bio.terra.service.iam.IamRole;
+import bio.terra.service.iam.IamService;
+import bio.terra.service.resourcemanagement.DataLocationService;
+import bio.terra.service.resourcemanagement.google.GoogleResourceService;
+import bio.terra.service.snapshot.Snapshot;
+import bio.terra.service.snapshot.SnapshotDataProject;
+import bio.terra.service.snapshot.SnapshotService;
+import bio.terra.service.snapshot.exception.SnapshotNotFoundException;
+import bio.terra.stairway.FlightContext;
+import bio.terra.stairway.Step;
+import bio.terra.stairway.StepResult;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.UUID;
+
+public class DeleteSnapshotAuthzBqAcls implements Step {
```

Comment:
It looks like the convention is for `Step` subclasses to end in `Step`.

Reply:
Huh, interesting.  I happened to base this off of the one case that doesn't do this :)